### ISSUE TITLE
Design | Profile | Notification settings | Design System Council (DSC) review

### ISSUE THREAD

**Issue by andaleliz** [department-of-veterans-affairs/va.gov-team#55895]
## Background

We [submitted an update](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1646) to a design system component and [requested an async review](https://dsva.slack.com/archives/C01DBGX4P45/p1680026072351329). Now we need to follow along with the review process.  The design work being reviewed was completed in #54046 

## Tasks
- [x] Participate in async review
- [x] Make any recommended edits/improvements
- [ ] Prep for FE

## Acceptance Criteria
- [x] we have a decision from the DSC
- [ ] Ready for FE, if approved

**Comment by andaleliz** [comment]
@BerniXiongA6 FYI, created this for the next sprint

**Comment by andaleliz** [comment]
Hey @mtcA6! They did meet on 4/7, and did not give clear feedback.  Their decision was "They can use it but it isn’t the right interaction design". I'm not sure what the "right" interaction design is and am waiting to hear back via the comments in [this ticket](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1646).

>do I need to chase the DSC to get that feedback?

It is frustrating that I continue to have to follow up with them, but we don't _need_ to know for any business reason at the moment. It's not solving an urgent a11y issue that needs to be remedied right away.

They meet every two weeks. Maybe they can carve out some time to discuss again and provide details async? I can check about that. Unfortunately, I have a conflict during their meeting time tomorrow, and I'll be out of town the next time they meet on 5/5 so I won't be able to go in person.

**Comment by mtcA6** [comment]
Would you want me to go to the meeting on 5/5? @andaleliz I'm happy to try to get us the right feedback

**Comment by andaleliz** [comment]
Followed up with the team again [via Slack](https://dsva.slack.com/archives/C0NGDDXME/p1681839486561709). 

@mtcA6 if they don't respond, and they hold the meeting as scheduled on 5/5, I think it would be helpful for you to go and try to get clarity on this. Otherwise I suppose our team can just move forward with the "wrong" interaction, but that feels bad to me.

**Comment by andaleliz** [comment]
@mtcA6 I understood Matt's example more to be an example of a pattern that to be deprecated because it had accessibility problems, rather than him referencing it because he felt like that was a better choice for our particular use case.   The wizard pattern he mentioned is really only similar to ours in that it dynamically changes the UI on the page and introduces alerts/content based on a users interaction. Our component was designed w/ accessibility specialists and has also been tested by assistive tech users, so I feel confident that it _won't_ fall into the same defect categories as the thing Matt referenced.

**Comment by andaleliz** [comment]
Hi @adamwhitlock1 - these changes are approved by the design system council, and I've updated [the spec doc](https://www.sketch.com/s/afd69a1f-72d2-430b-9b62-285e9d3f479c/v/LaqObl/a/9P4QlQn/) with the latest changes.

The summary of the changes is that we're:

- updating the success icon
- increasing top/bottom margin on the alert
- increasing padding in the alert
- updating the width so it's the width of the content, vs the container

Do you need any other documentation from me to implement these changes? Thanks!

**Comment by andaleliz** [comment]
Thanks @adamwhitlock1! I added the saving state the specs, and will close this out. Will you tag me on the FE ticket so I can follow along? I'm excited too!
